sys/net/gnrc/netif/Kconfig

```diff
@@ -22,6 +22,7 @@ config GNRC_NETIF_MSG_QUEUE_SIZE_EXP
 
 config GNRC_NETIF_IPV6_ADDRS_NUMOF
     int "Maximum number of unicast and anycast addresses per interface"
+    default 3 if DHCPV6_CLIENT_ADDRS_NUMOF != 0
```

Comment:
This is not exposed to Kconfig. Rather go with `DHCPV6_CLIENT_LEASE_MAX` here (as defined [here](https://github.com/RIOT-OS/RIOT/pull/16228/files#diff-c71df66f366e8e1d1752c9749926d33c11d9e4aa12576459668563dbc3f1a114R15))